conf.d/fzf.fish

```diff
@@ -4,6 +4,7 @@ if not set --query fzf_fish_custom_keybindings
     bind \cf '__fzf_search_current_dir'
     bind \cr '__fzf_search_history'
     bind \cv '__fzf_search_shell_variables'
+    bind \cb '__fzf_search_abbreviations'
```

Comment:
I love how this keybinding is mnemonically inspired: b for 'bbreviations. However, it is found to move left. I wonder if people would miss using Ctrl+b to move left? Just a thought--not asking you to change it yet.